docs/adguard-for-android/solving-problems/multiple-user-profiles.md

```diff
@@ -9,9 +9,23 @@ This article covers AdGuard for Android, a multifunctional ad blocker that prote
 
 :::
 
+In this article you will find the methods on how to solve issues with AdGuard when you set up *multiple accounts* or *Restricted Profile*s on your android devices.
+
+## Problems caused by multiple user profiles
+
+On Android 9 and later, if AdGuard is installed for more than one user profile on your device, you may encounter problems when uninstalling the app. When you uninstall AdGuard from one user profile, the app will still appear on the app list, but you won't be able to uninstall or reinstall it. It happens because AdGuard is installed for a different user profile on the device.
```

Comment:
It happens because AdGuard is installed for a different user profile on the device.
может, потому что настроен другой пользовательский профиль (а не установлен)?

Reply:
It happens —> This happens

Reply:
It happens because AdGuard is installed for a different user profile on the device.
тут же про то, что AdGuard установлен в разных профилях?

Reply:
Artem Ivanov
:do-not-disturb:  [3:46 PM](https://performix.slack.com/archives/DRM4K21C3/p1689079607969319)
привет!
[3:51](https://performix.slack.com/archives/DRM4K21C3/p1689079871596059)
It happens because AdGuard is installed for a different user profile on the device.
может, потому что настроен другой пользовательский профиль (а не установлен)?
он был создан и настроен, если технически описывать, что произошло. Но вообще речь про то, что другой аккаунт просто существует
[3:51](https://performix.slack.com/archives/DRM4K21C3/p1689079891765709)
It happens because AdGuard is installed for a different user profile on the device.
тут же про то, что AdGuard установлен в разных профилях?
да, верно